pages/index.js

```diff
@@ -1,3 +1,61 @@
+import Image from "next/image";
+import { EffectCards } from "swiper";
+import { SwiperSlide } from "swiper/react";
+import Button from "../components/Button";
+import Carousel from "../components/Carousel";
+import Tile from "../components/Tile";
+
+const ROLE_TILE_SIZE = 300;
+const ROLES = ["Development", "Marketing", "Design"];
+
+const RoleTile = ({ role }) => {
+  return (
+    <div className="flex flex-col">
```

Comment:
What's this outer `div` doing? It doesn't look like it's doing anything new to me, right?